projects/plugins/jetpack/class.jetpack.php

```diff
@@ -7398,94 +7398,78 @@ public static function is_active_and_not_offline_mode() {
 	 */
 	public static function get_products_for_purchase() {
 		$products = array();
-		if ( ! is_multisite() ) {
```

Comment:
Does this mean that backups can now work on Multisite networks, or should this remain?

Reply:
> Does this mean that backups can now work on Multisite networks, or should this remain?

Even though, technically, I think we now offer a download link instead of an actual restore (I think this is only for individual sites on a multisite installation and not the network), then no. Multisite Networks are not officially supported :)
The main reason I'm removing it here is because if someone links to Jetpack Backup's product description, then I still want to show it.

I few options if we want to change it back:

* We create a fallback page for "dead" product description links that gives a generic message like e.g.: "Product is not available on your site" if a multisite site ends up on e.g. the backup product description.
* We rename all references from "products for purchase" to "product descriptions" to avoid any confusions here if the listed products are available for the site itself.

I would like the product description page to be as "dumb" as possible and just show the description if the user ends up on the "page". Then it would be the place that implements the link that is responsible for if it should be shown or not.

Context: I basically took the "products for purchase" and repurposed it since it was completely deprecated anyway (unused, fatal error query component, ignored for lints etc) and a lot of the logic could be reused for a new use-case and in my mind it still made sense with "for purchase" since it's basically sales texts.

Reply:
> I would like the product description page to be as "dumb" as possible and just show the description if the user ends up on the "page". Then it would be the place that implements the link that is responsible for if it should be shown or not.

That makes sense. Let's leave it as it is then, that seems okay as long as we're mindful of that.

> Then it would be the place that implements the link that is responsible for if it should be shown or not.

Do you think we could add a comment on the method mentioning that somehow? Would that be overkill?